tests/NuGet.IndexingTests/TestSupport/MockTopDocs.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Lucene.Net.Search;
+
+namespace NuGet.IndexingTests.TestSupport
+{
+    public class MockTopDocs : TopDocs
+    {
+        public MockTopDocs(int totalHits, ScoreDoc[] scoreDocs, float maxScore) :
+            base(totalHits, scoreDocs, maxScore)
```

Comment:
I think it would be trivial to switch to a "Mock" class when the need arises, so for now we can just get rid of it.